angular/base/src/polyfills.ts

```diff
@@ -55,6 +55,7 @@
 /***************************************************************************************************
  * Zone JS is required by default for Angular itself.
  */
+import './zone-flags.ts';
```

Comment:
Very minor point but maybe above the separator on l.55 is better? It would immediately follow the instruction on zone-flags.ts that way.

Reply:
Agreed. thanks! https://github.com/ionic-team/starters/pull/750